gem/templates/comments/comment.html

```diff
@@ -22,25 +27,26 @@
   {% if node.is_removed %}
     <p>{% trans "This comment has been removed by the community. Please read our platform rules." %}</p>
   {% else %}
-
     {% if truncate_comment %}
       <p>{{ node.comment|truncatechars:60 }}</p>
     {% else %}
       <p>{{ node.comment }}</p>
     {% endif %}
-
     <span class="date">{{node.submit_date|timesince}} {% trans "ago" %}</span>
     {% if not node.user|is_in_admin_group %}
       <a href="{% url 'report_comment' node.pk %}" class="report">{% trans "Report" %}</a>
     {% endif %}
-
     {% if node.parent == None %}
-      {% if request.user.is_authenticated and request.user|is_in_admin_group %}
-        <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="report">
-          {% trans "Reply"%}
-        </a>
+      {% if request.user.is_authenticated %}
+        <div class="comment__reply-meta">
+          <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="comment__reply-meta-link">
+            {% trans "Reply"%}
+          </a>
+          <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="comment__reply-meta-link comment__reply-meta-replies-list">
+            {% trans "Replies"%}
```

Comment:
There should be a count for a the number of replies here, see https://github.com/praekelt/molo.commenting/pull/42/files#diff-4f03d5f24d520b64a1d3e834956f0d05R38